pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -234,16 +272,24 @@ func (exp *flowExporter) sendTemplateSet(templateSet ipfix.IPFIXSet, templateID
 	}
 
 	// Get all elements from template record.
-	exp.elementsList = elements
+	if !isIPv6 {
+		exp.elementsListv4 = elements
+	} else {
+		exp.elementsListv6 = elements
+	}
 
 	return sentBytes, nil
 }
 
-func (exp *flowExporter) sendDataSet(dataSet ipfix.IPFIXSet, record flowexporter.FlowRecord, templateID uint16) error {
+func (exp *flowExporter) sendDataSet(dataSet ipfix.IPFIXSet, record flowexporter.FlowRecord, templateID uint16, isIPv6 bool) error {
```

Comment:
Do we need the args `templateID` and `isIPv6`? I think`record flowexporter.FlowRecord` and `exp *flowExporter` should have these already.

Reply:
Yes. Removed.